src/modules/fancyzones/editor/FancyZonesEditor/App.xaml.cs

```diff
@@ -20,6 +21,7 @@ public partial class App : Application
         public App()
         {
             ZoneSettings = new Settings();
+            System.Threading.Thread.CurrentThread.CurrentUICulture = CultureInfo.CurrentUICulture;

```

Comment:
@alekhyareddy28 Why is this needed?

Reply:
This is used to get the culture of the user and choose the corresponding resource file.

Reply:
Yes @ryanbodrug-microsoft. You are right. I shall remove that statement as it is redundant.

"If you do not explicitly set the current UI culture and you do not define a default culture for the current app domain, the CultureInfo.CurrentUICulture property is set implicitly by the Windows GetUserDefaultUILanguage function. This function is provided by the Multilingual User Interface (MUI), which enables the user to set the default language. If the UI language is not set by the user, it defaults to the system-installed language, which is the language of operating system resources." 

Reference - https://docs.microsoft.com/en-us/dotnet/api/system.resources.resourcemanager?view=netframework-4.8#resourcemanager-and-culture-specific-resources